### ISSUE TITLE
Symbolic linked files are not listed when by PowerColorLS

### ISSUE THREAD

**Issue by Drllap** [gardebring/PowerColorLS#10]
Script I used for testing:
```
mkdir test_folder
echo "some test text" >> test_file.txt
cd test_folder
New-Item -Name the_link -ItemType SymbolicLink -Value ../test_file.txt
PowerColorLS
Get-ChildItem
```

Calling `PowerColorLS` doesn't output anything:
![image](https://user-images.githubusercontent.com/3659986/140199220-438f8bdf-503d-4186-a71d-4f7f64a8fc95.png)

`Get-ChildItem` output:
![image](https://user-images.githubusercontent.com/3659986/140199424-95607381-58aa-4ad6-865b-a2d6f39d3f09.png)

Modules and versions:
![image](https://user-images.githubusercontent.com/3659986/140199574-4467c4db-44b4-4163-beee-36412cdc3986.png)

Using:

- Theme = honukai
- microsoft-windows-terminal 1.10.2714.0
- PowerShell 5
- Windows 10

**Comment by Drllap** [comment]
Looks like this only happens when the symbolic link is the only thing in the folder

**Comment by Drllap** [comment]
I found the issue [here](https://github.com/gardebring/PowerColorLS/blob/606e1fe867b03b3bcd0f7f45a71806bb8b79d3f2/src/Private/FileAndFolderFunctions.ps1#L44)

If there is only one item in the path `Get-ChildItem` returns `FileInfo`. The `Length` member of `FileInfo` is the file size which is zero in the case of symbolic links.

A potential fix would just be to delete this if statement.

**Comment by gardebring** [comment]
Thank you @Drllap for your issue report and your excellent description including reproduction steps.
I will look into this when I have a bit of time over :)

**Comment by Drllap** [comment]
I have created a pull request that fixes this: #11

**Comment by gardebring** [comment]
Hi @Drllap . Really appreciate the PR! However, I had already fixed this in a different branch, just didn't get around to merge and actually deploy it up...
Now version 1.0.4 is in the main branch and also deployed to Powershell Gallery. 
Please let me know if there are any issues still :)
